hardware/raspberrypi/bcm2711_bootloader_config.md

```diff
@@ -268,20 +298,24 @@ XHCI_DEBUG=0x3
 Default: `0x0` (no USB debug messages enabled)  
 
 ## config.txt - configuration properties
+
+<a name="bootloader_flags"></a>
 ### boot_load_flags
```

Comment:
Should the anchor be `"boot_load_flags"` ?

Reply:
done